### PR TITLE
SC-5479 - direct resources linking

### PR DESCRIPTION
**Pull request by BlackGhostHD**
# Description

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: src/components/organisms/AddContentButton.vue [hunk 1]
```diff
@@ -66,7 +67,10 @@ export default {
 			return response && response.status === 201;
 		},
 		getUrl() {
-			return `/content/${this.resource.ref.id}`;
+			return (
+				getMetadataAttribute(this.resource.properties, "ccm:wwwurl") ||
```
Comment (mwolowyk-cap) [inline src/components/organisms/AddContentButton.vue:71]: According to the content team and due to CORS restrictions it should be: 

`getMetadataAttribute(this.resource.properties, "cclom:location");`

File: src/components/organisms/ContentCard.vue [hunk 2]
```diff
@@ -94,25 +95,17 @@ export default {
...
 			);
 			return provider ? provider.replace("/n", "").trim() : "Schul-Cloud";
 		},
 		thumbnail() {
 			return (
-				this.getMetadataAttribute(
-					this.resource.properties,
-					"ccm:thumbnailurl"
-				) || this.resource.preview.url
+				getMetadataAttribute(this.resource.properties, "ccm:thumbnailurl") ||
```
Comment (mwolowyk-cap) [inline src/components/organisms/ContentCard.vue:107]:
```suggestion
				
```

Comment (mwolowyk-cap) [reply src/components/organisms/ContentCard.vue:107]: We should access `this.resource.preview.url` due to CORS

Comment (BlackGhostHD) [reply src/components/organisms/ContentCard.vue:107]: @mwolowyk-cap  Please change this in your PR

Comment (mwolowyk-cap) [reply src/components/organisms/ContentCard.vue:107]: ok

File: src/components/organisms/AddContentButton.vue (hunk 1 above)
Comment (BlackGhostHD) [reply src/components/organisms/AddContentButton.vue]: done

**Comment by mwolowyk-cap** [comment]
Was already integrated in https://github.com/schul-cloud/nuxt-client/pull/1124/files

**Pull request closed without merging**